cypress/utils/utils.ts

```diff
@@ -1124,6 +1124,7 @@ export function getRowsAmount(): number {
 
 export function getRandomApplicationData(
     appName?,
+    tags?,
     options?: { sourceData?; binaryData? }
```

Comment:
When adding new parameters to an existing method, add them at the end of the method even if the parameter is optional. Adding this new parameter in the middle will make every test (+70 tests) that was passing the `options` parameter to fail.